src/tests/class_tests/openms/source/FileHandler_test.cpp

```diff
@@ -218,7 +218,7 @@ TEST_REAL_SIMILAR(exp[2][1].getPosition()[0], 130)
 TEST_REAL_SIMILAR(exp[2][2].getPosition()[0], 140)
 
 tmp.getOptions() = PeakFileOptions();
-TEST_EQUAL(tmp.loadExperiment(OPENMS_GET_TEST_DATA_PATH("MzMLFile_1.mzML"), exp), true)
+TEST_EQUAL(tmp.loadExperiment(OPENMS_GET_TEST_DATA_PATH("MzMLFile_1.mzML"), exp, {FileTypes::MZML}, OpenMS::ProgressLogger::NONE, true, true), true)
```

Comment:
TEST_TRUE

Reply:
CTest tells me to use TEST_EQUAL: "Boolean test macro. This macro tests if its argument evaluates to 'true'. If possible use TEST_EQUAL(a, b) instead of TEST_TRUE(a==b), because the latter makes bug tracing harder."